src/components/accordion/__tests__/CdrAccordion.spec.js

```diff
@@ -1,212 +1,158 @@
 import { shallowMount, mount } from '../../../../test/vue-jest-style-workaround.js';
```

Comment:
Some general formatting changes made here:

All expects statements should correspond to an it block
All test setup should not be in an it block, but in the before hook of a describe block.
Use nested describes when it makes sense